.github/workflows/release-apidocs.yml

```diff
@@ -0,0 +1,140 @@
+on:
+  push:
+    branches:
+      - mainnet
+      - testnet
+      - integrationnet
+      - develop
+  pull_request:
+
+name: Publish API docs
+
+env:
+  GITHUB_TOKEN: ${{ github.token }}
+  AWS_REGION:
+    "us-west-1"
```

Comment:
probably should be taken from env/secret

Reply:
will keep it hardcoded for now